Sources/StreamVideo/StreamVideo.swift

```diff
@@ -217,18 +124,26 @@ public class StreamVideo {
             defaultAPI.middlewares.append(anonymousAuth)
         }
         self.prefetchLocation()
+        connectTask = Task {
+            if user.type == .guest {
+                do {
+                    let guestInfo = try await loadGuestUserInfo(for: user, apiKey: apiKey)
+                    self.user = guestInfo.0
```

Comment:
nit: Not in love with the parameter index for access

Reply:
fixed